### PR TITLE
Make the appRoot param optional

### PR DESCRIPTION
**Pull request by j-f1**
I haven’t tested this yet, but the docs say it should work.

### CODE REVIEW

**Comment by j-f1** [comment]
@ArtOfCode- It’s based on the latest commit. You might be looking at the `master` branch on my fork. I’m PRing from `patch-1`.

**Comment by ArtOfCode-** [comment]
@j-f1 I'm looking at the changes tab on the PR. I'll put some line comments on so you can see what I mean.

File: docs/getting-started.md [hunk 1]
```diff
@@ -84,12 +81,12 @@ The third step in creating this app is to create a controller to handle the rout
...
 
 ```js
-exports.index = function (req) {
-  this.render({view: true, locals: {ip: req.connection.remoteAddress, ua: req.headers['user-agent']}});
+exports.index = function (req, render) {
+  render({view: true, locals: {ip: req.connection.remoteAddress, ua: req.headers['user-agent']}});
 };
 
-exports.json = function (req) {
-  this.render({json: {status: 'up', ip: req.connection.remoteAddress, ua: req.headers['user-agent']}});
```
Comment (ArtOfCode-) [inline docs/getting-started.md]: This form is correct for 0.3.0 - no `render` parameter for controller actions, and `render` prefixed with `this.`

Comment (j-f1) [inline docs/getting-started.md]: Fixed. Sorry about that.

**Pull request merged**